src/components/TxNotification.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import { abbreviateString } from 'utils'
+import { Network } from 'types'
+import { useWalletConnection } from 'hooks/useWalletConnection'
+
+interface TxNotificationProps {
+  txHash: string
+}
+
+function getDomainPrefix(networkId: Network): string {
+  return !networkId || networkId === Network.Mainnet ? '' : Network[networkId].toLowerCase() + '.'
+}
+
+export const TxNotification: React.FC<TxNotificationProps> = ({ txHash }: TxNotificationProps) => {
+  const { networkId } = useWalletConnection()
+  const href = `https://${getDomainPrefix(networkId)}etherscan.io/tx/${txHash}`
+
+  return (
+    <div>
+      The transaction has been sent! Check{' '}
+      <a href={href} target="_blank" rel="noopener noreferrer">
```

Comment:
What about this being a component too?
Props:
* type: 'tx' (default) | 'address' | 'code'
* tx: string
* label?: string (by default uses the abbreviated tx, but we can use it for example for the contract code link in the deposit widget)

Reply:
Check the latest revision :+1: